Bottom: f1b7e21968beea0d56af76b4f20cd640b851e0b5
Top:    a8bebc02722556c48ac1bf19e88dafbe5d1d0284
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-20 18:05:09 +0100

Refresh of tidy-up-pad-drawing-routines

---

diff --git a/src/crosshair.c b/src/crosshair.c
index a3d5da7..64cf020 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -256,7 +256,7 @@ XORDrawElement (ElementTypePtr Element, LocationType DX, LocationType DY)
   PAD_LOOP (Element);
   {
     if (PCB->InvisibleObjectsOn ||
-        (TEST_FLAG (ONSOLDERFLAG, &moved_pad) != 0) == Settings.ShowSolderSide)
+        (TEST_FLAG (ONSOLDERFLAG, pad) != 0) == Settings.ShowSolderSide)
       {
         /* Make a copy of the pad structure, moved to the correct position */
         PadType moved_pad = *pad;
